Anchor.toml

```diff
@@ -47,7 +47,7 @@ program = "./tests/deps/spl_token_2022.so"
 mmm = "mmm3XBJg5gk8XJxEKBvdgptZz6SgK4tXvn36sodowMc"
 
 [scripts]
-test = "npx ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts"
+test = "npx jest"
```

Comment:
You dont need `npx` if you have it as a dependency in the package, you can just do `npm`, should be faster